backend/danswer/server/manage/administrative.py

```diff
@@ -166,10 +166,14 @@ def create_deletion_attempt_for_connector_id(
         get_editable=True,
     )
     if cc_pair is None:
+        error = (
```

Comment:
adding more logging (previously would not give any details in the logs)

Reply:
I have been doing the same thing. 

However, this is just a consequence of the recent logging rework causing failed API calls to not show any error beyond whatever 400 code. We should probably look into solving that...